src/core/tests/visitors/op/paged_attention.cpp

```diff
@@ -0,0 +1,62 @@
+    const auto value_cache = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, ov::Shape{10, 4, 10, 4});
+    const auto past_lens = std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::Shape{4});
+    const auto subsequence_begins = std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::Shape{5});
+    const auto block_indices = std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::Shape{10});
+    const auto block_indices_begins = std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::Shape{5});
+    const auto scale = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, ov::Shape{});
+    const auto sliding_window = std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::Shape{});
+    const auto alibi_slopes = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, ov::Shape{4});
+    const auto max_context_len = std::make_shared<ov::op::v0::Parameter>(ov::element::i32, ov::Shape{});
+
+    const auto paged_attention = std::make_shared<ov::op::PagedAttentionExtension>(query,
+                                                                                   key,
+                                                                                   value,
+                                                                                   key_cache,
+                                                                                   value_cache,
+                                                                                   past_lens,
+                                                                                   subsequence_begins,
+                                                                                   block_indices,
+                                                                                   block_indices_begins,
+                                                                                   scale,
+                                                                                   sliding_window,
+                                                                                   alibi_slopes,
+                                                                                   max_context_len);
+    NodeBuilder builder(paged_attention,
+                        {query,
+                         key,
+                         value,
+                         key_cache,
+                         value_cache,
+                         past_lens,
+                         subsequence_begins,
+                         block_indices,
+                         block_indices_begins,
+                         scale,
+                         sliding_window,
+                         alibi_slopes,
+                         max_context_len});
+    auto g_paged_attention = ov::as_type_ptr<ov::op::PagedAttentionExtension>(builder.create());
+
+    constexpr auto expected_attr_count = 1;
+    EXPECT_EQ(builder.get_value_map_size(), expected_attr_count);
```

Comment:
This test for attribute visitor has been added in this PR and it is failing, because the `PagedAttentionExtension` class doesn't have `visit_attributes` method. It should be implemented to make this test passing. 
But again, let's keep it only if crucial for the reference impl and aim to reduce unrelated changes in this PR.

Reply:
Oki, removed!